vnext/Folly/TEMP_UntilFollyUpdate/json.cpp

```diff
@@ -14,1101 +14,1104 @@
-        } else if (s[0] == '\"') {
-          state = State::None;
-        }
-        result.push_back(s[0]);
-        break;
-      case State::InlineComment:
-        if (s.startsWith('\n')) {
-          // preserve the line break to preserve the line count
-          result.push_back(s[0]);
-        } else if (s.startsWith("*/")) {
-          state = State::None;
-          ++i;
-        }
-        break;
-      case State::LineComment:
-        if (s[0] == '\n') {
-          // preserve the line break to preserve the line count
-          result.push_back(s[0]);
-          state = State::None;
-        }
-        break;
-      default:
-        throw std::logic_error("Unknown comment state");
-    }
-  }
-  return result;
-}
-
-} // namespace json
-
-//////////////////////////////////////////////////////////////////////
-
-dynamic parseJsonWithMetadata(StringPiece range, json::metadata_map* map) {
-  return parseJsonWithMetadata(range, json::serialization_opts(), map);
-}
-
-dynamic parseJsonWithMetadata(
-    StringPiece range,
-    json::serialization_opts const& opts,
-    json::metadata_map* map) {
-  json::Input in(range, &opts);
-
-  uint32_t n = in.getLineNum();
-  auto ret = parseValue(in, map);
-  if (map) {
-    map->emplace(&ret, json::parse_metadata{{{0}}, {{n}}});
-  }
-
-  in.skipWhitespace();
-  if (in.size() && *in != '\0') {
-    in.error("parsing didn't consume all input");
-  }
-  return ret;
-}
-
-dynamic parseJson(StringPiece range) {
-  return parseJson(range, json::serialization_opts());
-}
-
-dynamic parseJson(StringPiece range, json::serialization_opts const& opts) {
-  json::Input in(range, &opts);
-
-  auto ret = parseValue(in, nullptr);
-  in.skipWhitespace();
-  if (in.size() && *in != '\0') {
-    in.error("parsing didn't consume all input");
-  }
-  return ret;
-}
-
-std::string toJson(dynamic const& dyn) {
-  return json::serialize(dyn, json::serialization_opts());
-}
-
-std::string toPrettyJson(dynamic const& dyn) {
-  json::serialization_opts opts;
-  opts.pretty_formatting = true;
-  opts.sort_keys = true;
-  return json::serialize(dyn, opts);
-}
-
-//////////////////////////////////////////////////////////////////////
-// dynamic::print_as_pseudo_json() is implemented here for header
-// ordering reasons (most of the dynamic implementation is in
-// dynamic-inl.h, which we don't want to include json.h).
-
-void dynamic::print_as_pseudo_json(std::ostream& out) const {
-  json::serialization_opts opts;
-  opts.allow_non_string_keys = true;
-  opts.allow_nan_inf = true;
-  out << json::serialize(*this, opts);
-}
-
-void PrintTo(const dynamic& dyn, std::ostream* os) {
-  json::serialization_opts opts;
-  opts.allow_nan_inf = true;
-  opts.allow_non_string_keys = true;
-  opts.pretty_formatting = true;
-  opts.sort_keys = true;
-  *os << json::serialize(dyn, opts);
-}
-
-//////////////////////////////////////////////////////////////////////
-
-} // namespace folly
+       firstEsc += prefix;
```

Comment:
Did the indentation on these files actually change?

Reply:
fixed! I usually just copy/paste these files from github to upgrade and then yarn format will fix the indentations, it looks like yarn format doesn't apply to this folder